server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPServlet.java

```diff
@@ -90,6 +90,15 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
         }
     }
 
+    private void writeValue(HttpServletResponse response, List<Object[]> responses) {
+        try {
+            objectMapper.writeValue(response.getOutputStream(), responses);
+        } catch (IOException e) {
+            LOGGER.info("Error handling request", e);
```

Comment:
is it the request hadling that fails or the response send ? What can trigger this IOException ?